src/lib/containers/entity_finder/details/view/DetailsViewRow.tsx

```diff
@@ -79,24 +77,35 @@ export const DetailsViewRow: React.FunctionComponent<DetailsViewRowProps> = ({
     },
   )
 
+  // TODO: Handle pagination, preferably with an IntersectionObserver
+  // We can't use IntersectionObserver via ref on a div within a <select> input
+  // It might make more sense to use react-select here
```

Comment:
For `select` inputs, useInView won't work for populating options. This would be a good place to use react-select, the custom select component used in the new annotation editor, but that's a bigger change than I want to make right now.